data/fixtures/scopes/swift/index.json

```diff
@@ -0,0 +1,3 @@
+{
+  "imports": ["swift"]
```

Comment:
Remove this file. This import means that swift is importing swift. It's only used when one language is a superset of another language.

Reply:
thank you. I removed the file. still the same error.